sdk/__test__/util.spec.ts

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
```

Comment:
lets follow the existing unit test convention i.e. keep the unit test in the same folder as the file being tested (not in a new __test__ folder)

Reply:
Updated